README.md

```diff
@@ -6,8 +6,7 @@ This project is a playground to play with Kubernetes.
 
 ## Components
 
-1. "Controller" VM: a Vagrant box running Docker where we run an Ansible
-   instance to configure the whole environment
+1. "Controller": a Docker container where we run an Ansible instance to configure the whole environment
```

Comment:
The Controller is actually a separate VM, that runs a containerized Ansible instance. We proposed to get rid of this in #79 and run the Ansible container on one of the cluster nodes (but we didn't do it yet).

Reply:
> The Controller is actually a separate VM, that runs a containerized Ansible instance
sure, as I mentioned in issue #100 
my point is that the important message is that we have a cointainer running ansible, it is not important that we have a VM... if we change like I propose we do not have to change anything later on because the new sentence is valid now and it will be valid when we move the container into the master... we could also have the container in the host... I mean it does not matter where it is... 
if we want to keep the information of where this container is currently running we can add
"Controller": a Docker container where we run an Ansible instance to configure the whole environment (this container is now running on a separate VM, we plan to run it in the Kubernetes Master VM)
Note also that the term VM is currently used only for the controller, while is not used for the Kubernetes Master and Minions, we should harmonize this in any case...

Reply:
OK!